recipes/cairo/all/conanfile.py

```diff
@@ -146,18 +154,20 @@ def _configure_autotools(self):
         if self._autotools:
             return self._autotools
 
+        def boolean(value):
+            return "yes" if value else "no"
```

Comment:
yes_no lambda was more elegant I believe, why this change?

Reply:
Rejected by the linter ( [PEP-8](https://peps.python.org/pep-0008/#programming-recommendations) - E731 in Flake8 ).

As for the naming, I think `boolean` is a better name for a function than `yes_no`.